src/js/control-bar/progress-control/seek-bar.js

```diff
@@ -169,12 +198,6 @@ class SeekBar extends Slider {
    *          The current percent at a number from 0-1
    */
   update(event) {
-    // if the offsetParent is null, then this element is hidden, in which case
-    // we don't need to update it.
-    if (this.el().offsetParent === null) {
```

Comment:
This code never actually runs, since the control bar is always visible for screen readers.